parquet/README.md

```diff
@@ -0,0 +1,34 @@
+# Parquet Sample data
```

Comment:
Parquet files have not been moved to LFS in this PR -- which seems to be the intent of the PR.
1) I'm only seeing readme files changed.  Proper LFS setup should show all of the parquet files transitioning to LFS.
2) There needs to be a commit of the `.gitattributes` (?) file so that future files or file changes are properly handled.